atlp-rwanda/ecommerce-app-predators-bn
# #18-ft-wishlist

**what does this PR do?**

a post request to add an item to the wishlist

**Description of Task to be completed?**

- The item should be added to the buyer's wishlist
- The product details should be retrieved from the database
-  The updated wishlist should be stored in the buyer's account A confirmation message should be sent to the frontend to be displayed to the buyer

  **How should this be manually tested?**

- Checkout this branch.
- Run npm install to install all dependencies.
- Run npm run dev
- Check the console output for test results.

Trello Card : (https://trello.com/c/7b3NFqBd/18-a-buyer-should-be-able-to-add-a-product-to-wish-list-and-view-the-list)
